ephios/user_management/forms.py

```diff
@@ -256,3 +266,21 @@ def __init__(self, *args, **kwargs):
     form=QualificationGrantForm,
     extra=0,
 )
+
+
+class WorkingHourRequestForm(Form):
+    when = DateField(widget=CustomDateInput)
+    hours = DecimalField()
+    reason = CharField()
+
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop("request")
+        super().__init__(*args, **kwargs)
+
+    def create_consequence(self):
+        WorkingHoursConsequenceHandler.create(
+            user=self.request.user,
+            when=self.cleaned_data["when"],
```

Comment:
You are upcasting a date to datetime here. Do we want to store start times or is the date and hours enough?

Reply:
That was kind of a quick fix because I couldn't get our split input to work. 
I'm not sure whether the time is necessary. At the moment, it's not displayed anywhere and I think that the overview would be a bit crowded with all the times